initdb.py

```diff
@@ -0,0 +1,5 @@
+from mailbbgun.models import db
+
+db.create_all()
+
+print("mailbbgun database schema initialized.")
```

Comment:
I prefer out-of-band migrations.  Although I didn't do it for this project, my reasoning for it is that I would prefer to have very narrow least-privilege database roles.  So the credentials used in the API process should not have permission to modify schema, and the credentials used to migrate the schema should not be allowed to add users, etc.  I think I have this point of view from years of hanging out with hardcore security guys.